app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,24 @@
+class PostsController < ApplicationController
+  def show
+    @post = Post.find(params[:id])
```

Comment:
Why use the show page here? Doesn't it make sense to be redirected to the index and have the post be displayed there with the other posts?

This is also a bit premature optimization, you could argue that my suggestion of showing the created post in the index action instead is also that. But I assume that you will abandon that when you create the forum model and that the user creates the post inside of that.

Reply:
I agree

Reply:
We wanted the visitor to be redirected to the show view of the post so he can proof read it and then only after that go to the forum view, read Julie's comment below.